### PR TITLE
Bump docker to 10.0.2

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: nightly/docker-compose.yml
Comment (onurctirtir) [inline nightly/docker-compose.yml:2]: I couldn't understand the difference between those two

File: Dockerfile-alpine
Comment (onurctirtir) [inline Dockerfile-alpine:22]: I wonder why we didn't hit to that issue when building nightly docker images ?

Comment (onurctirtir) [reply Dockerfile-alpine:22]: also, should we add those two to `Dockerfile` too ?

File: Dockerfile
Comment (onurctirtir) [inline Dockerfile:1]: let's bump it to 13.2

Comment (onurctirtir) [reply Dockerfile:1]: and have a changelog entry for that, right after `Bump Citus version to 10.0.2` :


>\* Bump PostgreSQL version to 13.2

File: Dockerfile-alpine
Comment (onurctirtir) [inline Dockerfile-alpine:1]: similar to https://github.com/citusdata/docker/pull/212/files#r587341039
let's use 13.2 here

File: nightly/docker-compose.yml
Comment (onurctirtir) [reply nightly/docker-compose.yml:2]: if there is not difference at all, please revert the changes in this particular file

File: Dockerfile-alpine
Comment (gurkanindibay) [reply Dockerfile-alpine:22]: alpine has the smallest set of all packages compared to other base images. So it means that others already include that packages. Or we need to add these packages if they are needed on runtime . So after testing we may add these packages to other docker files

Comment (onurctirtir) [reply Dockerfile-alpine:22]: I understood the reason. This is because, we directly install citus from packages in other docker images but we build citus from source for alpine, so we need those dependencies.

But I'm still wondering this:
>I wonder why we didn't hit to that issue when building nightly docker images ?

File: Dockerfile
Comment (gurkanindibay) [reply Dockerfile:1]: Fixed

File: Dockerfile-alpine
Comment (gurkanindibay) [reply Dockerfile-alpine:1]: Fixed

File: CHANGELOG.md
Comment (onurctirtir) [inline CHANGELOG.md:4]: we need a new line between them (e.g as we did in v9.5.0), maybe I should have wrote this more clear, sorry

File: Dockerfile-alpine
Comment (onurctirtir) [inline Dockerfile-alpine:22]: Should we have lz4 & zstd ? Other than that, all LGTM

**Review by onurctirtir — COMMENTED** [review]
https://github.com/citusdata/docker/blob/release-10.0.2-1614851010/Dockerfile-alpine#L28

let's not forget to change this line to `./configure --with-security-flags &&`

File: Dockerfile-alpine
Comment (onurctirtir) [inline Dockerfile-alpine:23]: let's remove them from here
```suggestion
```

File: Dockerfile-alpine
Comment (onurctirtir) [inline Dockerfile-alpine:47]:
```suggestion
RUN apk add postgresql-libs zstd zstd-libs lz4-dev lz4
```

File: Dockerfile-alpine
Comment (onurctirtir) [inline Dockerfile-alpine:25]:
```suggestion
    apk add --no-cache libcurl zstd zstd-libs lz4-dev lz4 zstd-dev && \
```

**Review by onurctirtir — COMMENTED** [review]
Please apply all changes from https://github.com/citusdata/docker/commit/20f7e53a9ea88419bb93604376a15c2df3af8341 before merging

**Review by onurctirtir — APPROVED** [review]
Approving **conditionally**.

Please apply all changes from https://github.com/citusdata/docker/commit/20f7e53a9ea88419bb93604376a15c2df3af8341 before merging.
Please also make sure that ci outputs are fine for alpine and also make sure that we can do `docker run` after making those changes

**Pull request merged**
